packages/charting/src/components/VerticalStackedBarChart/examples/VerticalStackedBarChart.Example.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import { VerticalStackedBarChart } from '@uifabric/charting';
+import { IVSChartDataPoint, IVerticalStackedChartProps } from '@uifabric/charting';
+import { DefaultPalette } from 'office-ui-fabric-react/lib/Styling';
+import { DirectionalHint } from 'office-ui-fabric-react';
+import { ChoiceGroup, IChoiceGroupOption } from 'office-ui-fabric-react/lib/ChoiceGroup';
+
+const options: IChoiceGroupOption[] = [
+  { key: 'singleCallout', text: 'Single callout' },
+  { key: 'MultiCallout', text: 'Stack callout' },
+];
+
+interface IVerticalStackedBarState {
+  width: number;
+  height: number;
+  selectedCallout: string;
+}
+
+export class VerticalStackedBarChartCalloutExample extends React.Component<{}, IVerticalStackedBarState> {
```

Comment:
Can you name the file to match the component (see other examples here)?

Reply:
Changes done